/**:
  ros__parameters:
    occlusion_spot:
      pedestrian_vel: 1.0                  # [m/s] assume pedestrian is dashing from occlusion at this velocity
      safety_time_buffer: 0.1              # [s] delay for the system response
      threshold:
        detection_area_length: 100.0        # [m] the length of path to consider perception range
        stuck_vehicle_vel: 1.0              # [m/s] velocity below this value is assumed to stop
        lateral_distance: 1.5               # [m] maximum lateral distance to consider hidden collision
      # road type parameters
      motion:
        safety_ratio: 0.8                   # [-] jerk/acceleration ratio for safety
        v_min: 1.0                          # [m/s] minimum velocity allowed
        a_min: -1.0                         # [m/s^2] minimum deceleration for safe deceleration.
      sidewalk:
        min_occlusion_spot_size: 1.0     # [m] occupancy grid must contain an UNKNOWN area of at least size NxN to be considered a hidden obstacle.
        slice_size: 1.5     # [m] size of sidewalk slices in both length and distance relative to the ego trajectory.
        focus_range: 3.0            # [m] buffer around the ego path used to build the sidewalk area.
      grid:
        free_space_max: 40  # maximum value of a free space cell in the occupancy grid
        occupied_min: 60   # minimum value of an occupied cell in the occupancy grid
